examples/streaming/kafka-hdfs-pipeline/scripts/createtopic

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
```

Comment:
how about getting these scripts into a README as they could not be generally applied ?